hack/lima-and-qemu.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+set -eux -o pipefail
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+
+# shellcheck disable=SC2024
+sudo -b opensnoop >"${DIR}/opensnoop.log" 2>/dev/null
+trap 'sudo pkill dtrace' EXIT
+
+# Capture both legacy firmware (alpine) and UFI (default) usage
+for EXAMPLE in default alpine; do
+	limactl delete -f "${EXAMPLE}"
+	limactl start --tty=false "${DIR}/examples/${EXAMPLE}.yaml"
+	limactl shell "${EXAMPLE}" uname
+	limactl stop "${EXAMPLE}"
+	limactl delete "${EXAMPLE}"
+done
+
+sudo pkill dtrace
+
+"${DIR}/hack/lima-and-qemu.pl" opensnoop.log
```

Comment:
Can we combine sh and pl in a single script? If not, can we have more descriptive file names to distinguish two?

Reply:
I've moved the shell logic into the Perl script and deleted the shell one.

I've also changed it so that you can run `./hack/lima-and-qemu.pl alpine` and only include the alpine example, so you can save another 2MB in the tarball by not including the bloated UFI firmware. 😸